juju/juju
# Add patches to builds

## Description of change
This adds support for applying the patches during snap creation, or via a local build using the makefile. This effectively deprecates the old python script. The script has been left in for now until the CI build process has transitioned away from using it.

## QA steps
Build juju with make build and snapcraft. Ensure the patches are applied as part of the process.
